readme.md

```diff
@@ -179,6 +179,40 @@ alfy.output([{
 
 <img src="media/screenshot-output.png" width="694">
 
+Using the `variables` property:
```

Comment:
I think we should put this under it's own section so we can easily link it from the highlights and to make it more clear. Not sure where though, any ideas @sindresorhus? Maybe a new section `Environment variables` under `Usage` or `Example`? Or on the same level as those?